openrtb3/no_bid_status_codes.go

```diff
@@ -0,0 +1,55 @@
+package openrtb3
+
+// NonBidStatusCode represents OpenRTB Loss Reason Code enumeration.
+//
+// It lists the options for an exchange to inform a bidder as to the reason why they did not win an item.
+type NonBidStatusCode int64
+
+// NonBidStatusCode options.
+//
+// Values 500+ are exchange specific values; should be communicated with buyers beforehand.
+const (
+	NoBidWinningBid                         NonBidStatusCode = -1  // Winning Bid
+	NoBidGeneral                            NonBidStatusCode = 0   // No Bid - General
+	NoBidInternalTechnicalError             NonBidStatusCode = 1   // No Bid - Internal Technical Error
+	NoBidInvalidRequestObj                  NonBidStatusCode = 2   // No Bid - Invalid Request
+	NoBidKnownWebCrawler                    NonBidStatusCode = 3   // No Bid - Known Web Crawler
+	NoBidSuspectedNonHumanTraffic           NonBidStatusCode = 4   // No Bid - Suspected Non-Human Traffic
+	NoBidClouDataCenterOrProxyIP            NonBidStatusCode = 5   // No Bid - Cloud, Data Center, or Proxy IP
+	NoBidUnsupportedDeviceObj               NonBidStatusCode = 6   // No Bid - Unsupported Device
+	NoBidBlockedPublisherOrSite             NonBidStatusCode = 7   // No Bid - Blocked Publisher or Site
+	NoBidUnmatchedUserReq                   NonBidStatusCode = 8   // No Bid - Unmatched User
+	NoBidDailyUserCapMet                    NonBidStatusCode = 9   // No Bid - Daily User Cap Met
+	NoBidDailyDomainCapMet                  NonBidStatusCode = 10  // No Bid - Daily Domain Cap Met
+	NoBidAdstxtAuthorizationUnavailable     NonBidStatusCode = 11  // No Bid - Ads.txt Authorization Unavailable
+	NoBidAdstxtAuthorizationViolation       NonBidStatusCode = 12  // No Bid - Ads.txt Authorization Violation
+	NoBidAdscertAuthenticationUnavailable   NonBidStatusCode = 13  // No Bid - Ads.cert Authentication Unavailable
+	NoBidAdscertAuthenticationViolation     NonBidStatusCode = 14  // No Bid - Ads.cert Authentication Violation
+	NoBidInsufficientAuctionTime            NonBidStatusCode = 15  // No Bid - Insufficient Auction Time
+	NoBidIncompleteSupplychain              NonBidStatusCode = 16  // No Bid - Incomplete SupplyChain
+	NoBidBlockedSupplyChainNodeObj          NonBidStatusCode = 17  // No Bid - Blocked SupplyChain Node
+	NoBidGeneralError                       NonBidStatusCode = 100 // Error - General
```

Comment:
Any difference between "NoBidGeneralError" and "NoBidGeneral"

Reply:
@ashishshinde-pubm  as per the Prebid Proposal
`NoBidGeneral ` (`0`)  => Auction ran successfully without demand for the impression.
`NoBidGeneralError` (`100`) => Technical problem occurred during the auction.
Even I am not very much clear here on this interpretation. We can ask this to Prebid